### PR TITLE
adds get_ledgers

### PR DESCRIPTION
**Pull request by hajdbo**
### Description:
Ledgers were missing from the REST api: https://docs.bitfinex.com/reference#rest-auth-ledgers

### Breaking changes:
- None

### New features:
- async def get_ledgers(self, symbol, start, end, limit=25, category=None):
        """
        Get all ledgers on account associated with API_KEY - Requires authentication.
        See category filters here: https://docs.bitfinex.com/reference#rest-auth-ledgers
        @return Array <models.Ledger>
        """

### Fixes:
- None

### PR status:
- [x] Version bumped
- [x] Change-log updated

### CODE REVIEW

**Comment by hajdbo** [comment]
It's been almost a month. Is there anything that should be changed?

File: bfxapi/rest/bfx_rest.py [hunk 1]
```diff
@@ -365,6 +365,21 @@ async def get_funding_credit_history(self, symbol, start, end, limit=25):
         credits = await self.post(endpoint, params=params)
         return [FundingCredit.from_raw_credit(c) for c in credits]
 
+    async def get_ledgers(self, symbol, start, end, limit=25, category=None):
+        """
+        Get all ledgers on account associated with API_KEY - Requires authentication.
+        See category filters here: https://docs.bitfinex.com/reference#rest-auth-ledgers
+        @return Array <models.Ledger>
+        """
+        endpoint = "auth/r/ledgers/{}/hist".format(symbol)
```
Comment (JacobPlaster) [inline bfxapi/rest/bfx_rest.py:374]: You can also emit the 'symbols' param  in order to receive data for all symbols, would be nice to accommodate this

**Comment by JacobPlaster** [comment]
Nice job @hajdbo

File: bfxapi/rest/bfx_rest.py (hunk 1 above)
Comment (JacobPlaster) [reply bfxapi/rest/bfx_rest.py:374]: Added in https://github.com/bitfinexcom/bitfinex-api-py/commit/1812942c5fa7770e3fb8089288950a5b8da2d53f

**Pull request merged**